src/sentry/auth/helper.py

```diff
@@ -969,3 +970,24 @@ def disable_2fa_required(self) -> None:
             event=audit_log.get_event_id("ORG_EDIT"),
             data={"require_2fa": "to False when enabling SSO"},
         )
+
+
+@transaction.atomic
+def EnablePartnerSSO(provider_key, org_member, sentry_org, provider_config):
```

Comment:
This file is mostly what we pass to the view. Since this is purely backend thing want to move it to a different helper. 
Alternatively add more comments.

Reply:
I'll add a docstring to explain this helper better 👍 

> This file is mostly what we pass to the view
Not sure i understand. Are you suggesting this helper will be utilized mainly by a view?
For now this is the most proper location for this helper, since this is a simplified abstraction from the AuthHelper.

Reply:
Being used by view is what the current documentation of the class is saying. 
`class AuthHelper(Pipeline):
    """
    Helper class which is passed into AuthView's`